qa/standalone/ceph-helpers.sh

```diff
@@ -1998,7 +1998,7 @@ function run_in_background() {
     shift
     # Execute the command and prepend the output with its pid
     # We enforce to return the exit status of the command and not the sed one.
-    ("$@" |& sed 's/^/'$$': /'; return "${PIPESTATUS[0]}") >&2 &
+    ("$@" |& sed 's/^/'$BASHPID': /'; return "${PIPESTATUS[0]}") >&2 &
```

Comment:
Weird $$ has always been the shell pid.

Reply:
But in this case it's the PID of the wrong process. It was a surprise for me too...